app/controllers/articles_controller.rb

```diff
@@ -219,6 +221,14 @@ def handle_user_or_organization_feed
     end
   end
 
+  def handle_latest_feed
+    page = params[:page].try(:to_i) || 1
+    ids_of_articles_on_latest = Articles::Feeds::LargeForemExperimental.new(page: page, number_of_articles: 12)
+      .latest_feed.includes([:user])
+      .includes(:taggings).map(&:id)
+    Article.find(ids_of_articles_on_latest)
```

Comment:
`Articles::Feeds::LargeForemExperimental.new.latest_feed` already returns article objects, so you don't need to re-fetch them with `Article.find`. On line 38 the action is rendering `@articles` so this should assign those articles to `@articles`

Let me know if anything is unclear!

Reply:
Hmm yes, I had to this trick since `ActiveModel::MissingAttributeError missing attribute: processed_html` error occurs on `feed.rss.builder` when `@articles` is the returned objects of `Articles::Feeds::LargeForemExperimental.new.latest_feed`

Here is that error line :eyes: 
```
#  app/view/articles/feed.rss.builder line 27
xml.description sanitize(article.processed_html, tags: allowed_tags, attributes: allowed_attributes)
```

Reply:
Ah got it! That's because the RSS feed needs a different set of fields than the `/latest` "human" page needs.

I wonder if we should simply separate the functionality instead of reusing `LargeForemExperimental`.

I looked at the code again, isn't what we need to return simply

https://github.com/forem/forem/blob/90aa6932040f584467c971456b56500e4d333c72/app/controllers/articles_controller.rb#L24

with the addition of `.where("score > ?", MINIMUM_SCORE_LATEST_FEED)` mutuated from `LargeForemExperimental` ?

In plain english: return all the published articles, with the attributes selected by the feed (`Article.feed`), in reverse chronological order of publication, filtering out those that have a score that's too low (so `MINIMUM_SCORE_LATEST_FEED`)

Does that make sense? 

I would add a test to make sure that's the case.

That's probably easier than recycling `LargeForemExperimental`, sorry for the confusion

Reply:
Yes, sure! I will update my PR accordingly :rocket: